spec/models/post_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  object = Post.new(title:"Title", description:"Description", user_id:1)
+
+  describe "title" do
+
+    it "has valid data" do
```

Comment:
BDD pasaulē testi stasta tev kas notiek noteiktajos gadijumos. Viemēr ir jautajums "jā izdarisi tā", tad būs "šis".
Testam vajag sevi turēt visu informāciju uzreiz, lai nevajadzētu liegt ar acim uz augšu vai factory failiem.

Pagaidam tev vajag notestet tiaki valīdacijas, un tu vari paskatities šo rakstu ar piemēriem:
https://semaphoreci.com/community/tutorials/how-to-test-rails-models-with-rspec